### ISSUE TITLE
0.14.1 QuantFig OHLC Candlesticks Missing;  X & Y Axis issues

### ISSUE THREAD

**Issue by coulanuk** [santosjorge/cufflinks#137]
First off, thanks guys for a great tool, appreciate your efforts.

The out of the box [Quantfig demo](https://github.com/santosjorge/cufflinks/blob/master/Cufflinks%20Tutorial%20-%20Quantfig.ipynb) does not behave as expected:

1. Plot does not display the price candlesticks; only the bands and mid line appear. Y-Axis has -ve price values.

2. If we remove the call to qf.add_bollinger_bands() the candlesticks display ok but the x axis dates appear as raw float values.  A work around for this is to convert the Dataframe datetime index to string, eg : `df.index = df.index.strftime('%Y-%m-%d')`.  Also Y axis values appear -ve.

My environment:
Cufflinks 0.14.1
Plotly 3.2.0. 
Jupyter on Ubuntu 18.04
Python 3.7.

```
# reproduce missing candlesticks on 0.14.1
import cufflinks as cf
cf.set_config_file(sharing='public',theme='pearl',offline=False)
cf.go_offline()
df=cf.datagen.ohlc()
qf=cf.QuantFig(df,title='First Quant Figure',legend='top',name='GS')
qf.add_bollinger_bands()
qf.iplot()
```
![newplot](https://user-images.githubusercontent.com/17710514/45484326-9d0b6500-b74b-11e8-8330-998279582abf.png)

![newplot 1](https://user-images.githubusercontent.com/17710514/45484687-e14b3500-b74c-11e8-86e3-5d9af03d29c8.png)

**Comment by santosjorge** [comment]
Thank you @coulanuk .
Fixed in 0.14.4

**Comment by coulanuk** [comment]
Thanks for the fix Jorge, I confirm it works well for me now.
